.snage.yaml

```diff
@@ -5,12 +5,15 @@ note:
 fields:
   - name: issue
     type: number
+    description: github issue number
   - name: type
     type: string
     enum: [ add, change, deprecate, remove, fix, security ]
+    description: one of [add, change, deprecate, remove, fix, security]
   - name: audience
     type: string
     enum: [ user, developer ]
+    description: one of [user, developer]
```

Comment:
I agree. But it could be helpful if `snage create --help` would also display the possible values for for every option. Snage itself could generate that from the field configuration though.